README.md

```diff
@@ -269,7 +269,16 @@ const info = {
 <!-- Follow Me -->
 #
 <div>
-<a href=" https://chat.whatsapp.com/KRqihLP7J7HEpM2dL6Zwt9" ><img src="https://img.icons8.com/color/48/000000/whatsapp--v1.png" alt="WhatsApp Icon" title="WhatsApp"/></a> <a href="https://t.me/joinchat/hP-WbTXCf185YTg1"><img src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" title="Telegram"/></a> <a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> <a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> <a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> <a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> <a href="https://www.linkedin.com/in/yoshitha-rathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> <a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
+<a href=" https://chat.whatsapp.com/KRqihLP7J7HEpM2dL6Zwt9" ><img src="https://img.icons8.com/color/48/000000/whatsapp--v1.png" alt="WhatsApp Icon" title="WhatsApp"/></a> 
+<a href="https://t.me/joinchat/hP-WbTXCf185YTg1"><img src="https://img.icons8.com/fluency/48/telegram-app.png" alt="Telegram Icon" title="Telegram"/></a> 
+<a href="https://github.com/YoshithaRathnayake"><img src="https://img.icons8.com/fluency/48/000000/github.png" alt="GitHub Icon" title="GitHub"/></a> 
+<a href="https://gitlab.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/gitlab.png" alt="GitLab Icon" title="GitLab"/></a> 
+<a href="https://twitter.com/Yoshitha2008"><img src="https://img.icons8.com/color/48/000000/twitter-circled--v1.png" alt="Twitter Icon" title="Twitter"/></a> 
+<a href="https://www.instagram.com/YoshithaRathnayake"><img src="https://img.icons8.com/color/48/000000/instagram-new.png" alt="Instagram Icon" title="Instagram"/></a> 
+<a href="https://www.linkedin.com/in/yoshitha-rathnayake"><img src="https://img.icons8.com/fluency/48/linkedin.png" alt="LinkedIn Icon" title="LinkedIn"/></a> 
+<a href="https://yoshitharathnayake.medium.com"><img src="https://img.icons8.com/ios-filled/48/ffffff/medium--v2.png" alt="Medium Icon" title="Medium"/></a>
+<a href="https://www.youtube.com/channel/UCiuDD_uA4v5PF2cr-3nTChA"><img width="48" height="48" src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```

Comment:
There is already given the size for the icon in the link so no need to add them manually right? @ad-astra-per-ardua 
What happened before you add the size? If there is no change by removing the width and the height remove them.

```suggestion
<a href="https://www.youtube.com/channel/UCiuDD_uA4v5PF2cr-3nTChA"><img src="https://img.icons8.com/color/48/youtube-play.png" alt="Youtube Icon"/></a>
```